config/default.py

```diff
@@ -57,6 +57,9 @@
 ASC_ATHLETES_API_KEY = 'secret'
 ASC_THIS_ACAD_YR = '2017-18'
 
+BOAC_API_KEY = 'Regents of the University of California'
+BOAC_CACHE_REFRESH_URL = 'https://ets-boac.example.com/api/refresh_me'
```

Comment:
The whole URL is stored in configs rather than just the hostname so that it's easy to switch between `/refresh`, `/import_refresh`, etc. as desired.

Reply:
Makes sense. In production, it seems likely that Nessie won't be sending this request until after Nessie has already taken over the import-from-Advisors-Community job, but this lets us test the dev-to-dev connection early.